libstarlark/src/main/java/net/starlark/java/eval/StarlarkThread.java

```diff
@@ -86,6 +91,54 @@ public void setMaxExecutionSteps(long steps) {
     this.stepLimit = steps;
   }
 
+  /**
+   *
+   * @return step limit
+   */
+  public long getStepLimit() {
+    return stepLimit;
+  }
+
+  /**
+   *
+   * @return expiration date in ms since 1970
```

Comment:
this is surprising to me. i thought ms would be relative not, absolute since 1970

Reply:
having it be absolute would mean a job could not be re-run without recomputing the value.